common/it/com/commercetools/sunrise/common/version/UserAgentIntegrationTest.java

```diff
@@ -0,0 +1,56 @@
+package com.commercetools.sunrise.common.version;
+
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.http.HttpClient;
+import io.sphere.sdk.http.HttpHeaders;
+import io.sphere.sdk.http.HttpRequest;
+import io.sphere.sdk.http.HttpResponse;
+import io.sphere.sdk.projects.queries.ProjectGet;
+import org.junit.Test;
+
+import javax.annotation.Nullable;
+import java.util.concurrent.CompletionStage;
+
+import static com.commercetools.sunrise.it.SphereClientFixtures.provideHttpClient;
+import static com.commercetools.sunrise.it.SphereClientFixtures.provideSphereClient;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class UserAgentIntegrationTest {
```

Comment:
And I've missed this too 😨 
I guess I was focusing too much in the code :P
So the test class should always have the same base name as the class it's testing (that helps a lot maintenance since you can quickly associate both classes). In this case `SunriseSolutionInfoIntegrationTest`.